source/voice_control/troubleshooting_the_s3_box.markdown

```diff
@@ -0,0 +1,67 @@
+---
+title: "Troubleshooting the ESP32-S3-BOX-3"
+---
+
+This section provides troubleshooting steps for the ESP32-S3-BOX-3 by Espressif.
+
+## Error: Unable to connect to Wi-Fi
+
+### Symptom
+
+The ESP32-S3-BOX-3 shows a message that it is unable to connect to Wi-Fi.
+
+### Remedy
+
+1. First, check if your network is ready in general.
+   1. Make sure your router is on and within reach.
+   2. Make sure you have chosen a Wi-Fi network that supports 2.4 GHz. The ESP32-S3-BOX-3 won't show up on a 5 GHz network.
+
+2. The next step is to make sure you entered the correct Wi-Fi password. Follow the steps either under **Option 1** or **Option 2**, depending on whether or not you have the ESPHome add-on installed.
+   - **Option 1**: You do not have the ESPHome add-on installed or you have the add-on but did **not** adopt the ESP32-S3-BOX-3. If the device is shown in green, it is not adopted. 
+      ![ESP32-S3-BOX-3 not adopted](/images/assist/esp32-not-adopted.png)
+     1. Make sure the USB cable is plugged into the ESP32-S3-BOX-3.
+     2. Go to https://esphome.io/projects/index.html select the **Connect** button, then **Change Wi-Fi**.
+   - **Option 2**: You already have the ESPHome add-on installed and adopted the ESP32-S3-BOX-3 on your ESPHome dashboard.
+     1. Make sure the USB cable is plugged into the ESP32-S3-BOX-3.
+     2. In Home Assistant, go to [**Settings** > **Add-ons** > **ESPHome**](https://my.home-assistant.io/redirect/supervisor_addon/?addon=5c53de3b_esphome), and **Open Web UI**.
+     3. On the **ESP32-S3-BOX-3** add-on, select edit.
+       ![ESP32-S3-BOX-3 open config file](/images/assist/esps32-s3-edit-config.png)
+       - **Result**: An editor opens, showing the configuration file.
+       ![ESP32-S3-BOX-3 edit config file](/images/assist/esp32-edit-wifi-credentials.png)
+     4. Under the Wi-Fi component, enter your Wi-Fi credentials:
+        - Instead of `!secret wifi_ssid`, enter the name of your 2.4&nbsp;GHz Wi-Fi.
```

Comment:
YOu may want to guide the users to check their secrets file before hardcoding the credential in the firmware.

It's located here:
![Screenshot 2023-12-13 at 14 35 12](https://github.com/home-assistant/home-assistant.io/assets/5878296/46415a36-53e6-4cd7-a3de-b7e4c430d0d4)


 It is not that hardcoding the credential is less secure.. It is just that firmware files tend to be shared, and so it is better to sanitize it.
 Also, if the user really changed the Wifi password, changing it on the secrets file will fix all ESPHome devices at once.